ipi/inputs/normalmodes.py

```diff
@@ -159,6 +159,24 @@ class InputNormalModes(Input):
                 "options": ["exact", "cayley", "bab"],
             },
         ),
+        "fft_threads": (
+            InputValue,
+            {
+                "dtype": int,
+                "default": 1,
+                "help": "The number of threads to be used for the FFT.",
+                "dimension": None,
+            },
+        ),
+        "fft_float32": (
```

Comment:
Again, should we use the faster default?

Reply:
as above, I think it was a bad choice to make float32 default, the speedup is not massive and the increase in conserved quantity drift noticeable